renderdoccmd/renderdoccmd_android.cpp

```diff
@@ -28,6 +28,7 @@
 #include <string.h>
 #include <unistd.h>
 #include <string>
+#include "../os/android/android_specific.h"
```

Comment:
`renderdoccmd` can only access the core library through the existing replay public interface, not through directly including files like this. Likewise this is something very specific that I don't want to tunnel through and expose up to a cross-platform external interface - I'm not sure why the implementation can't just be in this file though, since it's android specific and fairly small. See below comment though.